.gitignore

```diff
@@ -0,0 +1,17 @@
+node_modules
+.DS_Store
+package-lock.json
+package.json
```

Comment:
This one is not good 😄 Same for `index.js`. Maybe you meant to not add them in this commit? Then you can use `git add FILE_NAME` to selectively include or a GUI like the Github App. Some editors also have features that allow you to commit visually. I use the one in Atom which let's me select specific lines of files to commit so I can logically break down changes that belong together, but are made across files.